drivers/generic_gpio_common.h

```diff
@@ -0,0 +1,48 @@
+#ifndef GENERIC_GPIO_COMMON_H
+#define GENERIC_GPIO_COMMON_H
+
+#include <stdlib.h>
+#include <gpiod.h>
```

Comment:
This is technically `#if HAVE_GPIOD_H` or `#if WITH_LIBGPIOD` - right?
The file claims to be common, but hardcodes a dependency on (currently the only) one backend here.